package.nls.json

```diff
@@ -251,6 +266,26 @@
     "AWS.placeHolder.selectProfile": "Select a credential profile",
     "AWS.profile.recentlyUsed": "recently used",
     "AWS.running.command": "Running command: {0}",
+    "AWS.s3.createBucket.placeHolder": "Bucket Name",
+    "AWS.s3.createBucket.prompt": "Create Bucket",
+    "AWS.s3.createBucket.error.illegalCharacters": "Bucket name must only contain lowercase letters, numbers, hyphens, and periods",
+    "AWS.s3.createBucket.error.invalidEnd": "Bucket name must end with a lowercase letter or number",
+    "AWS.s3.createBucket.error.invalidLength": "Bucket name must be between 3 and 63 characters long",
+    "AWS.s3.createBucket.error.invalidStart": "Bucket name must start with a lowercase letter or number",
+    "AWS.s3.createBucket.error.misusedPeriods": "Periods in bucket name must be surrounded by a lowercase letter or number",
+    "AWS.s3.createBucket.error.resemblesIpAddress": "Bucket name must not resemble an IP address",
```

Comment:
I feel like we already have these lines of text for the deploy workflow? If so, no point in duplicating them

Reply:
Will consolidate.